admin/resources/js/components/AuthContainer.tsx

```diff
@@ -44,7 +44,12 @@ export const AuthContainer: React.FC = ({ children }) => {
     const queryParams = parseUrlQueryParameters(location);
     const accessToken = queryParams.access_token;
     const refreshToken = queryParams.refresh_token;
-    if (accessToken && queryParams.token_type === "Bearer") {
+    if (
+      accessToken &&
+      queryParams.token_type.localeCompare("Bearer", "en", {
+        sensitivity: "base",
+      }) === 0
+    ) {
```

Comment:
This change will conflict with #1196.  This change is simpler than Tristan's and his handles the case insensitive to this one should probably be removed.

Reply:
OK, I merged in the newer AuthContainer.tsx file to resolve the conflict.